f/connectors/alerts/alerts_gcs.py

```diff
@@ -124,221 +131,251 @@ def _get_tif_rel_filepath(blob_name, local_file_path, territory_id):
     Returns
     -------
     str
-        The relative file path for the TIF file.
+        The relative file path for the file.
     """
-    year = local_file_path.split("/")[-3]
-    month = local_file_path.split("/")[-2]
-    filename = local_file_path.split("/")[-1]
+    path_parts = Path(local_file_path).parts
+    year = path_parts[-3]
+    month = path_parts[-2]
+    filename = path_parts[-1]
     alert_id = filename.split(".")[0].split("_")[-1]
-    filepath = os.path.join(str(territory_id), year, month, alert_id, "images")
-    return filepath
+    filepath = Path(str(territory_id), year, month, alert_id)
+    if filename.lower().endswith((".tif", ".tiff", ".jpg", ".jpeg")):
+        filepath = filepath / "images"
+    return str(filepath)
 
 
 def sync_gcs_to_local(
-    dst_path, storage_client, bucket_name, territory_id, alerts_metadata_filename
+    destination_path,
+    storage_client,
+    bucket_name,
+    territory_id,
+    alerts_metadata_filename,
 ):
     """Download files from a GCS bucket to a local directory.
 
     Parameters
     ----------
-    dst_path : str
+    destination_path : str
         The local directory where files will be downloaded.
     storage_client : google.cloud.storage.Client
     bucket_name : str
         The name of the GCS bucket to download from.
     territory_id : int
-        The path prefix for which files shuold be downloaded.
+        The path prefix for which files should be downloaded.
     alerts_metadata_filename : str
-        An additional file to download: mean tto be the filename containing alerts metadata.
+        An additional file to sync from the GCS bucket.
 
     Returns
     -------
-    set of str
-        the (local) file names that were written.
+    tuple
+        A tuple containing a set of str with the local file names that were written
+        and the alerts metadata content.
```

Comment:
While it's correct that it's a tuple, numpy doc convention would split out the multiple return elements and document each:

```suggestion
    geojson_files : set of str
        local file names of the geojson files that were written
    tiff_files : set of str
        local file names of the TIFF files that were written
    alerts_metadata : str
        alerts metadata content.
```